eudi/credentials/sdjwtvc/crypto.go

```diff
@@ -17,13 +17,13 @@ type JwtCreator interface {
 }
 
 type DefaultEcdsaJwtCreator struct {
-	key *ecdsa.PrivateKey
+	PrivateKey *ecdsa.PrivateKey
```

Comment:
Instead of exporting the private key, it might be better to use a builder pattern (so you can get the key in, but never out)

Reply:
Ah, accidentally exported the variable indeed.
Fixed.